### PR TITLE
Calculate unbound period for undelegated value

### PR DESCRIPTION
**Pull request by danielailie**

### CODE REVIEW

File: react-delegationdashboard/src/helpers/contractDataDefinitions.ts [hunk 1]
```diff
@@ -61,13 +62,32 @@ export class Stats {
...
+  ) {
     this.topUpFactor = topUpFactor;
+    this.roundDuration = roundDuration;
+    this.roundsPerEpoch = roundsPerEpoch;
+    this.roundsPassedInCurrentEpoch = roundsPassedInCurrentEpoch;
     this.topUpRewardsGradientPoint = topUpRewardsGradientPoint;
   }
 }
 
+export class NetworkStatus {
```
Comment (bogdan-rosianu) [inline react-delegationdashboard/src/helpers/contractDataDefinitions.ts:84]: just a question: why do we have this `NetworkStatus` that only contains `roundsPassedInCurrentEpoch` while we already have this metric in `NetworkConfig` ?

File: react-delegationdashboard/src/pages/Dashboard/PendingUndelegated/index.tsx [hunk 2]
```diff
@@ -16,14 +16,34 @@ const UndelegatedListView = () => {
     return denominate({ denomination, decimals, input: value, showLastNonZeroDecimal: false });
   };
 
+  const getTimeLeft = (value: QueryResponse, index: number, timeLeft: number) => {
+    let roundCurrentEpoch = networkConfig.roundsPerEpoch - networkConfig.roundsPassedInCurrentEpoch;
```
Comment (bogdan-rosianu) [inline react-delegationdashboard/src/pages/Dashboard/PendingUndelegated/index.tsx:20]: optional: can you rename variable to `roundsRemainingInEpoch`?

**Review by bogdan-rosianu — DISMISSED** [review]

File: react-delegationdashboard/src/helpers/contractDataDefinitions.ts (hunk 1 above)
Comment (danielailie) [reply react-delegationdashboard/src/helpers/contractDataDefinitions.ts:84]: I'll remove it

File: react-delegationdashboard/src/pages/Dashboard/PendingUndelegated/index.tsx (hunk 2 above)
Comment (danielailie) [reply react-delegationdashboard/src/pages/Dashboard/PendingUndelegated/index.tsx]: done

File: react-delegationdashboard/src/helpers/contractDataDefinitions.ts (hunk 1 above)
Comment (danielailie) [reply react-delegationdashboard/src/helpers/contractDataDefinitions.ts]: done

**Review by bogdan-rosianu — APPROVED** [review]

**Review by bogdan-rosianu — DISMISSED** [review]

**Review by constantintovisi — DISMISSED** [review]

**Review by constantintovisi — APPROVED** [review]

File: react-delegationdashboard/src/pages/Dashboard/Actions/DelegateAction/DelegateModal.tsx [hunk 3]
```diff
@@ -52,17 +52,17 @@ const DelegateModal = ({ show, balance, handleClose, handleContinue }: DelegateM
           )}
           <Formik
             initialValues={{
-              amount: '10',
+              amount: '1',
```
Comment (bcosma) [inline react-delegationdashboard/src/pages/Dashboard/Actions/DelegateAction/DelegateModal.tsx:55]: This value can be configured.
Another option would be to take it from SC:
`{
    "scAddress": "erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqylllslmq6y6",
    "funcName": "getContractConfig"
}`

Returns the followin as a list: 
`NumContracts - total number of contracts deployed by the manager (big int)
LastAddress - last generated delegation SC address (base64)
MinServiceFee (big int)
MaxServiceFee (big int)
MinDeposit - minimum deposit when creating a new delegation SC (big int)
MinDelegationAmount - minimum value to be delegated (big int)`

**Review by bcosma — APPROVED** [review]

File: react-delegationdashboard/src/pages/Dashboard/Actions/DelegateAction/DelegateModal.tsx (hunk 3 above)
Comment (danielailie) [reply react-delegationdashboard/src/pages/Dashboard/Actions/DelegateAction/DelegateModal.tsx:55]: I will fix this in another PR :)

**Pull request merged**